tests/integration/suite/test_project_catalog.py

```diff
@@ -211,6 +212,59 @@ def test_user_addition_after_creating_project_catalog(admin_mc,
     wait_for_projectcatalog_template_to_be_deleted(client, catalog_name)
 
 
+def test_project_catalog_access_before_app_creation(admin_mc, admin_pc,
+                                                    remove_resource,
+                                                    user_factory):
+    ns = admin_pc.cluster.client.create_namespace(name=random_str(),
+                                                  projectId=admin_pc.
+                                                  project.id)
+    remove_resource(ns)
+    client = admin_mc.client
+
+    name = random_str()
+    new_project = client.create_project(name=random_str(), clusterId='local')
+    remove_resource(new_project)
+    wait_for_condition('InitialRolesPopulated', 'True', client, new_project)
+    new_project = client.reload(new_project)
+
+    project_name = str.lstrip(new_project.id, "local:")
+    catalog_name = project_name + ":" + name
+    url = "https://github.com/mrajashree/charts.git"
```

Comment:
Do we have a QA or test charts repo - we should have one

Reply:
no we don't, but yeah we should

Reply:
or we can use rancher charts repo itself

Reply:
few tests were using rancher charts repo, but there was once a change in rancher charts that removed a version and all these tests started failing.

Reply:
We do have one `DEFAULT_CATALOG = "https://github.com/rancher/integration-test-charts"`

Reply:
okay, I wasn't aware of this and have been using my fork for charts in tests. updated it